IBPSA/ThermalZones/ISO13790/Zone5R1C/ZoneHVAC.mo

```diff
@@ -0,0 +1,73 @@
+within IBPSA.ThermalZones.ISO13790.Zone5R1C;
+model ZoneHVAC "Thermal zone for HVAC based on 5R1C network"
+  extends Zone(capMas(C=buiMas.heaC*AFlo - VRoo*1.2*1014))
+                                                         annotation (Icon(
+        coordinateSystem(preserveAspectRatio=false)), Diagram(coordinateSystem(
+          preserveAspectRatio=false)));
+
+  replaceable package Medium = Modelica.Media.Interfaces.PartialMedium;
+  parameter Integer nPorts=0 "Number of fluid ports" annotation (Evaluate=true,
+      Dialog(
+      connectorSizing=true,
+      tab="General",
+      group="Ports"));
+  Modelica.Blocks.Interfaces.RealInput latGains "latent gains"
+    annotation (Placement(transformation(extent={{-180,-100},{-140,-60}})));
+  Modelica.Blocks.Math.Gain mWat_flow(k=1/h_fg)
+    "Water flow rate due to latent heat gain"
+    annotation (Placement(transformation(extent={{-40,110},{-20,130}})));
+  Fluid.MixingVolumes.MixingVolumeMoistAir       vol(
+    redeclare final package Medium = Medium,
+    energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial,
+    m_flow_nominal=1,
+    V=VRoo,
+    nPorts=nPorts)       "Air volume"
+              annotation (Placement(transformation(extent={{50,110},{70,130}})));
+  Modelica.Fluid.Vessels.BaseClasses.VesselFluidPorts_b ports_b[nPorts](redeclare
+      each final package
+                    Medium = Medium)
+    "fluid port for adding HVAC system"                         annotation (
```

Comment:
fix format so it is easier to read. E.g., use
```modelica
 ports_b[nPorts](
  redeclare each final package Medium = Medium)
    "Fluid port for adding HVAC system"
  annotation(...
```

Reply:
Done